knimin/templates/pm_create_plate.html

```diff
@@ -0,0 +1,119 @@
+{% extends logged_in_index.html %}
+{% block head %}
+
+
+<script type="text/javascript">
+
+  function add_study_options(selDom) {
+    var studies = {% raw [ [int(s[0]), s[1], s[2], s[3]] for s in studies] %};
+    $.each(studies, function(idx, study) {
+      $('<option>').attr('value', study[0]).html('(ID: ' + study[0] +') ' + study[1]).appendTo(selDom);
+    });
+  };
+
+  function submit_plate_creation() {
+    // Retrieve the values introduced by the user
+    var plateType = $('#plate-type-select').val();
+    var studies = [];
+    $.each($('.study-select'), function(idx, studySel) {
+      if (studySel.value !== '-1') {
+        studies.push(studySel.value);
+      }
+    });
+    var plateName = $('#plate-name').val().trim();
+
+    // Do some checks with the values introduced by the user
+    $('#studies-warning').hide();
+    $('#plate-name-warning').hide();
+
+    if (studies.length === 0) {
+      // The user didn't choose any study
+      $('#studies-warning').show();
+    }
+    if (plateName.length === 0) {
+      // The plate name is empty or is just spaces
+      $('#plate-name-warning').html('* Please introduce a plate name');
+      $('#plate-name-warning').show();
+    } else {
+      $.get('/pm_sample_plate/name_check?name=' + plateName, function(data){
```

Comment:
Added the space. I've tried multiple characters (such `/` `'` `~` `@` `) and I couldn't break it.